Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -237,6 +238,9 @@ static bool ShouldDisplayGameListItem(const GameListItem& item)
   if (!show_platform)
     return false;
 
+  if (item.GetPlatform() == DiscIO::Platform::WII_WAD && !IOS::ES::IsChannel(item.GetTitleID()))
+    return false;
```

Comment:
What do you think about putting the check in `GameListItem::IsValid` instead? It would have the same performance advantage as your current implementation.

You should probably make the same change to DolphinQt2 too.